pkg/git/git.go

```diff
@@ -160,9 +149,7 @@ func plumbCommand(cmd *exec.Cmd, channel ssh.Channel, sidechannel io.Writer) *sy
 	}()
 
 	cmd.Stdout = channel
-	cmd.Stderr = channel.Stderr()
-
-	return &wg
```

Comment:
Good catch on that STDOUT/STDERR issue.

Okay, now I remember how this code works. Your explanation makes sense to me. I think this is good.